README.md

```diff
@@ -0,0 +1 @@
+school-library-ruby
```

Comment:
- [ ] You've added a readme in your project but the Readme is not descriptive. Please make sure you have added a proper readme.